src/matrix/matrixbeta.jl

```diff
@@ -72,15 +72,13 @@ end
 #  Properties
 #  -----------------------------------------------------------------------------
 
-dim(d::MatrixBeta) = dim(d.W1)
-
 size(d::MatrixBeta) = size(d.W1)
 
-rank(d::MatrixBeta) = dim(d)
+rank(d::MatrixBeta) = rank(d.W1)
```

Comment:
Rank or side-dimension?

Reply:
I guess I assumed it would be better to forward the call to `W1` in case its samples are not of full rank. By construction that can't happen though, it seems. So I guess it's a matter of personal preference whether to use `rank(d.W1)` or e.g. `size(d, 1)`. What do you prefer?

Reply:
@matbesancon Do you have any preference?

```suggestion
rank(d::MatrixBeta) = size(d, 1)
```

The deprecations cause time outs in the Turing tests downstream, so it would be good to fix them :slightly_smiling_face: